src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -68,6 +68,7 @@ internal class FunctionInstanceLogger : IAsyncCollector<FunctionInstanceLogEntry
                 var tableProvider = LogFactory.NewLogTableProvider(client);
 
                 ILogger logger = loggerFactory.CreateLogger(ScriptConstants.LogCategoryHostGeneral);
+                logger.LogInformation("AzureWebJobsDashboard is configured");
```

Comment:
We all think this should say : `"Azure WebJobs Dashboard is enabled."`

Brett also says this should be debug and not information. We all agree.